pulser-core/pulser/noise_model.py

```diff
@@ -128,6 +132,8 @@ class NoiseModel:
     - **amplitude**: Gaussian damping due to finite laser waist and
       laser amplitude fluctuations. Parametrized by ``laser_waist``
       and ``amp_sigma``.
+    - **detuning**: Dephasing fluctuations, parametrized by
+      ``dephasing_sigma``.
```

Comment:
Sometimes you call it `dephasing_sigma`, other `detuning_sigma`. We should pick one, I think `detuning_sigma` might be better